galaxykit/collections.py

```diff
@@ -0,0 +1,32 @@
+"""
+Functions for managing collections. Currently only handles uploading test collections.
+"""
+
+from subprocess import run
+from orionutils.generator import build_collection
+
+
+def upload_test_collection(client, namespace=None, collection_name=None):
+    """
+    Uploads a test collection generated with orionutils
+    """
+    config = {"namespace": namespace or client.username}
+    if collection_name is not None:
+        config["name"] = collection_name
+    artifact = build_collection(
+        "skeleton",
+        config=config,
+        extra_files={"meta/runtime.yml": {"requires_ansible": "==2.10"}},
+    )
+    collection_path = artifact.filename
+    run_args = [
```

Comment:
We should port the collection upload code from iqe-automation-hub-plugin

Reply:
Done, please let me know what you think. Had to re-implement some functionality that those tests used that was actually imported from ansible.